src/components/images_uploading/DisplayImageList.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { fetchImages } from "./../../actions";
+
+class DisplayImageList extends Component {
+  componentDidMount() {
+    this.props.fetchImages();
+  }
+  render() {
+    const { images } = this.props;
+
+    return (
+      <ol>
+        {images.map(image => {
+          return (
+            <>
+              {/* Warning: Warning: Each child in a list should have a unique "key" prop.
+               */}
+              <li key={image.s3key}>
+                Lot: {image.lot}, Unit Number:
+                {image.unitNumber}, Product Descripion:
+                {image.productDescription}, s3key:{" "}
+                <a
+                  href={
+                    `${process.env.REACT_APP_BASEURL}/images/` + image.s3key

```

Comment:
This can be re-written as:
`${process.env.REACT_APP_BASEURL}/images/${image.s3key}`